stubs/redis/redis/client.pyi

```diff
@@ -569,12 +570,12 @@ class Redis(Generic[_StrType]):
     def zscan_iter(self, name, match=..., count=..., score_cast_func=...): ...
     def sadd(self, name: _Key, *values: _Value) -> int: ...
     def scard(self, name: _Key) -> int: ...
-    def sdiff(self, keys: _Key | Iterable[_Key], *args: _Key) -> Set[_Value]: ...
+    def sdiff(self, keys: _Key | Iterable[_Key], *args: _Key) -> builtins.set[_Value]: ...
     def sdiffstore(self, dest: _Key, keys: _Key | Iterable[_Key], *args: _Key) -> int: ...
-    def sinter(self, keys: _Key | Iterable[_Key], *args: _Key) -> Set[_Value]: ...
+    def sinter(self, keys: _Key | Iterable[_Key], *args: _Key) -> builtins.set[_Value]: ...
     def sinterstore(self, dest: _Key, keys: _Key | Iterable[_Key], *args: _Key) -> int: ...
     def sismember(self, name: _Key, value: _Value) -> bool: ...
-    def smembers(self, name: _Key) -> Set[_StrType]: ...
+    def smembers(self, name: _Key) -> builtins.set[_StrType]: ...
```

Comment:
These need to be `builtins.set` rather than simply `set`, as there is a method named `set` in this class, which causes a name clash.